docs/datasource_api_userguide.md

```diff
@@ -194,8 +210,8 @@ df.show()
 */
 
 // Common options can also be passed in,
-// e.g. spark.tispark.plan.allow_agg_pushdown, spark.tispark.plan.allow_index_read, etc.
-// spark.tispark.plan.allow_index_read is optional
+// for example, spark.tispark.plan.allow_agg_pushdown, spark.tispark.plan.allow_index_read, etc.
```

Comment:
Is this change necessary here?

Reply:
We avoid using these abbreviations in our official document such as `i.e.`, `e.g.`, `etc.`.